src/pages/Facility/settings/organizations/FacilityOrganizationIndex.tsx

```diff
@@ -227,7 +227,7 @@ export default function FacilityOrganizationIndex({
                         asChild
                       >
                         <Link
-                          href={`/departments/${org.id}`}
+                          href={`/departments/${org.id}/users`}
```

Comment:
shouldn't it auto-redirect to users as default tab? 🤔 

```suggestion
                          href={`/departments/${org.id}`}
```